src/components/Summary/AlterationSummary.vue

```diff
@@ -162,7 +162,11 @@
               @valid="setEffectiveDateValid($event)"
             />
 
-            <v-card flat class="px-16 pb-8 mt-n12" v-if="isFutureEffective && isEffectiveDateTimeValid">
+            <v-card
+              flat
+              class="px-16 pb-8 mt-n12"
+              id="effectiveness-text"
```

Comment:
I hate to pick on an id name, but this doesn't mean what you think it means.


![](https://media.tenor.com/images/445b202edf3d33469ecc5aac82c3d8f0/tenor.gif)

Reply:
So what does it really mean?

Reply:
"Effectiveness" is how effective something is.

"Effective date" is the date it is effective on.

Hope you're ok with the joke above :)

Reply:
sure! I like jokes (when I get them 🤣)

Reply:
I changed the id though...